.pre-commit-config.yaml

```diff
@@ -1,22 +1,27 @@
 # Install pre-commit hooks via `pip install pre-commit && pre-commit install`
 
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.2.0
-  hooks:
-  - id: check-yaml
-  - id: end-of-file-fixer
-  - id: trailing-whitespace
-- repo: https://github.com/psf/black
-  rev: 22.3.0
-  hooks:
-  - id: black
-- repo: https://github.com/asottile/pyupgrade
-  rev: v2.31.1
-  hooks:
-  - id: pyupgrade
-    args: [--py36-plus, --keep-mock]
-- repo: https://github.com/asottile/blacken-docs
-  rev: v1.12.1
-  hooks:
-  - id: blacken-docs
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+#  - repo: https://github.com/pycqa/isort
+#    rev: 5.5.2
+#    hooks:
+#      - id: isort
+#        args: ["--profile", "black", "--filter-files"]
```

Comment:
`isort`'s defaults are in conflict with our flake8's defaults, need to address this elsewhere.

Reply:
yeah i guess we can drop it for now... until we organize our lint tools